neusomatic/python/sequencing_features.py

```diff
@@ -0,0 +1,269 @@
+#!/usr/bin/env python3
```

Comment:
@msahraeian is this copied verbatim from SomaticSeq?

Reply:
@marghoob yes